### PR TITLE
readme: visually emphasize performance criteria difference 

### PR DESCRIPTION
**Pull request by jqnatividad**
unless of course, this was intentional for emphasis.

### CODE REVIEW

**Comment by BurntSushi** [comment]
It's not redundant. One refers to program compile times. The other refers to regex compile times. A clearer wording is welcome.

**Comment by jqnatividad** [comment]
My bad... perhaps a little visual cue to emphasize the differences?

File: README.md [hunk 1]
```diff
@@ -233,10 +233,10 @@ nebulous. While nebulous, it guides this crate's architecture and the sorts of
...
 statements that follow as a result of the goal to be "fast":
 
-* When given the choice between faster regex searches and faster Rust compile
-times, this crate will generally choose faster regex searches.
-* When given the choice between faster regex searches and faster regex compile
-times, this crate will generally choose faster regex searches. That is, it is
+* When given the choice between faster regex searches and faster _*Rust compile
+times*_, this crate will generally choose faster regex searches.
+* When given the choice between faster regex searches and faster _*regex compile
+times*_, this crate will generally choose faster regex searches. That is, it is
```
Comment (BurntSushi) [inline README.md:239]: Please wrap to 79 columns (inclusive).

**Review by BurntSushi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jqnatividad) [reply README.md]: I just removed the double emphasis and just italicized the differences to keep the current wrapping.

Comment (jqnatividad) [reply README.md]: italicize only, and removed bold styling to keep current line wrapping to 79 cols inclusive.

**Review by BurntSushi — APPROVED** [review]
All righty, LGTM.

**Pull request merged**